content/changelog/2024-04-11-clever-tools-3.5.2.md

```diff
@@ -39,9 +39,7 @@ If you want to check if an application will deploy as a `TASK` or the `REGULAR`
 ```
 clever env set CC_RUN_COMMAND "bash a_bash_script.sh"
 ```
-Clever Tasks will evolve with enhancements planed over the coming months. Feel free to tell us about your needs or ideas on this!
-
-* [Learn more about Clever Tasks](/doc/develop/tasks/)
+Clever Tasks will evolve with enhancements planed over the coming months. Feel free to tell us about your needs or ideas on this subject!
```

Comment:
Mmmh, either we remove this line entirely or we ignore the Vale warnings. I don’t see of any good ways to says this differently :shrug:

Reply:
We remove it. Not promises in the doc, just the existing state of the art.

Reply:
My bad, this is the changelog. It's not great to promise things in the changelog, but there is a certain tolerance when it's about WIP.

Reply:
Still we can say it evolves continuously, showing which direction?

Reply:
This is why it's ok to leave it. But better not committing changelogs and new doc [on a single PR](https://github.com/CleverCloud/documentation/blob/main/CONTRIBUTING.md#-when-submitting-a-pr)